src/pages/year-2025.astro

```diff
@@ -0,0 +1,60 @@
+---
+import {Picture} from 'astro:assets';
+import {currentLanguage, useTranslations} from '../i18n/utils';
+import Scroller from '../layouts/Scroller.astro';
+import {getEntry} from 'astro:content';
+
+const pageLang = currentLanguage(Astro.currentLocale);
+const t = useTranslations(pageLang);
+
+const title = `${t('retrospective')} 2025`;
+
+const entry = await getEntry('scroller', '2025');
+const chapters = entry?.data ?? [];
+
+const tt = {
+    de: {
+        subheader: 'Rückblick auf das Jahr 2025',
+        intro: `Das Jahr 2025 hat für uns spannende Projekte, technologische Fortschritte und einen guten Austausch gebracht.`,
+        scroll: 'Zum Beginnen nach unten scrollen ↓',
+        outro: 'Regt eines dieser Themen bei euch eine Projektidee an oder wollt ihr Details zu einem Punkt hören?<br>Dann meldet euch gerne bei uns!',
+        website: 'Besucht unsere Webseite',
+    },
+    en: {
+        subheader: '2025 Year in Review',
+        intro: `The year 2025 has brought us exciting projects, technological advancements, and great collaboration.`,
+        scroll: 'Scroll to begin ↓',
+        outro: 'Do any of these topics spark a project idea for you, or would you like to hear more details about a specific point?<br>Feel free to reach out to us!',
+        website: 'Visit Our Website',
+    },
+}[pageLang];
+---
+
+<Scroller title={title} description={title} chapters={chapters}>
+    <div id="header">
```

Comment:
der Header ist etwas langweilig, so mit schwarzen Hintergrund.
Ausserdem fänd ich noch so einen animierten Fringer witzig, für die Leute die nicht checken dass man eine Seite ohne Scrollbar scrollen kann. Ich weiß es steht das aber ob das alle lesen ist die Frage.

https://www.flaticon.com/free-animated-icon/swipe-down_10522220

Reply:
[cf3e86c](https://github.com/geo-engine/geoengine.de/pull/27/commits/cf3e86c8226e9e38cb3f87d691c1db6d13320b90)

Reply:
ich würde ihn in einer neuen Zeilen 10x so groß machen. so finde ich, sieht man ihn kaum.

geile lösung btw

Reply:
[3a4483f](https://github.com/geo-engine/geoengine.de/pull/27/commits/3a4483f00e9c31bb2b7a8e55c29994862e0948cc)